### PR TITLE
Remove tracking code

### PR DESCRIPTION
**Pull request by stefanknotzer**
Additional fixed some annoying warnings

### CODE REVIEW

**Comment by Murmele** [comment]
do you know why the linux build fails? Seems not to be code related but something with ninja

File: src/dialogs/ConfigDialog.cpp [hunk 1]
```diff
@@ -108,35 +108,42 @@ class GeneralPanel : public QWidget
...
-      mRepo.config().setValue("user.name", text);
+    connect(mName, &QLineEdit::textChanged, this, [this](const QString &text) {
+      git::Config config = mRepo.config();
+      config.setValue("user.name", text);
     });
 
-    connect(mEmail, &QLineEdit::textChanged, [this](const QString &text) {
-      mRepo.config().setValue("user.email", text);
+    connect(mEmail, &QLineEdit::textChanged, this, [this](const QString &text) {
+      git::Config config = mRepo.config();
```
Comment (Murmele) [inline src/dialogs/ConfigDialog.cpp:117]: why not using directly mRepo.config()?

**Review by Murmele — CHANGES_REQUESTED** [review]

File: src/dialogs/ConfigDialog.cpp (hunk 1 above)
Comment (stefanknotzer) [reply src/dialogs/ConfigDialog.cpp:117]: When using Qt Creator 6.0.0 (Windows 10):
ConfigDialog.cpp:120:7: Call to temporary is a no-op: git::Config::setValue [clazy-writing-to-temporary}
no-op
I wanted to get rid of these warnings...

Comment (Murmele) [reply src/dialogs/ConfigDialog.cpp:117]: ok

File: src/ui/RepoView.cpp [hunk 2]
```diff
@@ -2630,7 +2628,7 @@ void RepoView::ignore(const QString &name)
   if (!file.open(QFile::Append | QFile::Text))
     return;
 
-  QTextStream(&file) << name << endl;
+  QTextStream(&file) << name << "\n";
```
Comment (KOLANICH) [inline src/ui/RepoView.cpp:2631]: Probably unrelated changes like this one shoudn't have gone into this PR.

Comment (stefanknotzer) [reply src/ui/RepoView.cpp:2631]: That is like a very bad habit: annoying warnings need to be fixed. And also irritating whitespaces...
I am going to make another appointment with my therapist...

Comment (Murmele) [reply src/ui/RepoView.cpp:2631]: are there many of them? Maybe we can remove all of them in one merge request?

Comment (KOLANICH) [reply src/ui/RepoView.cpp:2631]: It is pointless to remove them when they have already landed into the history. I mean that when commits touch unrelared things it is hard to audit such commits.

Comment (Murmele) [reply src/ui/RepoView.cpp:2631]: I agree with you. So we have to be more carefully with it.

Comment (stefanknotzer) [reply src/ui/RepoView.cpp:2631]: I will support this policy on future pull requests.

**Pull request merged**